RELEASE-VERSION

```diff
@@ -0,0 +1 @@
+0.11.58+464
```

Comment:
It turns out that wasn't the issue. The wiki said to change a line about the version to a hard number caused the window to pop-up.